tcms/testcases/templates/testcases/get.html

```diff
@@ -83,6 +83,30 @@ <h2 class="card-pf-title" style="text-align: left">
                 <span class="fa fa-calendar"></span>{{ object.create_date }}
             </h2>
 
+            <h2 class="card-pf-title" style="text-align: left">
+                {% with dummy_setup_duration=3000 %}
+                <span class="fa fa-clock-o"></span>{% trans 'Setup duration' %}:
+                {{ dummy_setup_duration }}
+                {% endwith %}
+               <!-- TODO: filter for duration readability -->
```

Comment:
Remove the TODOs - will make the tools happy and the list of items is also available in the issue description.